buildSrc/src/main/groovy/org/elasticsearch/gradle/precommit/PrecommitTasks.groovy

```diff
@@ -109,4 +110,11 @@ class PrecommitTasks {
         }
         return checkstyleTask
     }
+
+    private static Task configureNamingConventions(Project project) {
+        if (project.sourceSets.findByName("test")) {
+            return project.tasks.create('namingConventionsCheck', NamingConventionsTask)
```

Comment:
nit: I think we should just call this `namingConventions`, since we don't add "check" to any of our other check tasks.